polling_stations/templates/api_docs/blueprints/finder.apibp

```diff
@@ -203,3 +205,76 @@ the following conditions can be observed in the response body:
 + Response 404 (application/json)
 
         { "detail": "Address not found" }
+
+## Group Local Elections May 2019
+
+On 2 May 2019, local elections take place in England and Northern Ireland.
+
+### Ballots
+
+Local council elections will be taking place in 250 local authorities in England. Some areas also have mayoral elections.
+
+The `ballots` array can be used to determine if a user has elections scheduled. If `ballots` is an empty array, there is no scheduled election. Where elections are scheduled, `ballots` may contain multiple objects. For example, a user may vote in a local council election and mayoral election on the same day.
```

Comment:
Worth adding something about no elections that we cover (we can't assert no elections). 

Also, do you want to get in to the business of recommending that users don't show data about polling location if we don't know about elections? These docs don't have a lot of opinions about _how_ to show the data, but we do have opinions about it.  I don't know how much we should talk about that here.